.github/workflows/run-k8s-tests-on-sev.yaml

```diff
@@ -0,0 +1,65 @@
+name: CI | Run kubernetes tests on SEV
+on:
+  workflow_call:
+    inputs:
+      registry:
+        required: true
+        type: string
+      repo:
+        required: true
+        type: string
+      tag:
+        required: true
+        type: string
+
+jobs:
+  run-k8s-tests:
+    strategy:
+      fail-fast: false
+      matrix:
+        vmm:
+          - qemu-sev
+    runs-on: sev
+    env:
+      KUBECONFIG: /etc/rancher/k3s/k3s.yaml
```

Comment:
IIUC @fidencio has installed Rancher on TDX machine already, and the idea is to not re-install k8s over and over again for each job. We probably should be doing the same thing for SEV, but perhaps install a vanilla Kubernetes (with kubeadm?)? Unless, @ryansavino , you don't care about what k8s flavor to use :)

Reply:
I installed a vanilla kubernetes instance via kubeadm, but I can change it to rancher if that is required.